.github/workflows/connector-node-integration.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         java: [ '11', '17' ]
-    name: Java ${{ matrix.java }}
```

Comment:
Seems the CI pipeline cannot detect the new name, let's revert this line and see the result.

Reply:
reverted ;)